src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,64 @@
+package duke.task;
+
+import java.util.Objects;
+
+/**
+ * Represents a task. Task will generally have a description and have a isDone state. Some subclasses
+ * will have other details such as time (e.g. Event, Deadline)
+ */
+abstract public class Task {
```

Comment:
Perhaps public can be shifted in front of abstract. Noticed this in a couple of areas as well.